### ISSUE TITLE
Nothing is beautiful and everthing hurts (all old builds broken)

### ISSUE THREAD

**Issue by atz** [projecthydra/active_fedora#940]
I went back and tried to rerun previously successful Travis builds on older branches.  They are all broken.  

Failures include:
- `5.x`
- `6.4-stable`
- `6.5-stable`
- `6.6-stable`
- `6-7-stable` (the one I am currently interested in)
- `7.x-stable`
- `8.0-stable` (this was only 6 months old!)

The basic problem is not accounting for dependencies adequately.  Pretending each of these branches was always going to track hydra-jetty master forever was bogus.  At least by the time they flipped into maintenance mode, they should have pegged an explicit jetty artifact to test against.

**Comment by jcoyne** [comment]
Yes, someone who cares about those old versions really should take over their maintenance. One possibility would be to contract with DCE to maintain those old versions. Would Stanford be willing to pay for such a service?

**Comment by cbeer** [comment]
Or, I'd add -- yes, our previous selves were not as smart as we could have been. I don't think we've ever articulated a project-wide policy for how we support older versions of code, and it's typically fallen on those that use said components to backport useful changes, fix bugs, keep them up to date, etc.

I think all you need to do to get the appropriate versio nof hydra-jetty is to add:

```ruby
ZIP_URL = 'https://github.com/projecthydra/hydra-jetty/archive/v7.2.0.zip'
```

to your Rakefile. It isn't pretty, but it seems to do the job. I'm sure plenty of people would gladly review pull requests and release new versions as needed.

**Comment by atz** [comment]
@cbeer: I already did that for the branch that I care about currently, but I found the problem extends to many branches.

Would it be too weird to build a placeholder gem that merely provided releases that correspond to hydra-jetty builds available?  The basic problem would have been solved with semver, but the dep being unmanaged by bundler made that impossible.  We could call it... `jettwrapper-wrapper`  :\
